examples/templates/partials/result.html

```diff
@@ -1,7 +1,18 @@
 Bravo!
 
-<button hx-get="{{ url_for('home') }}" hx-target="body">
+<a href="{{ url_for('home') }}">
```

Comment:
This is one reason why I’m drawn away from htmx: paradoxically, it doesn’t seem to care much about Web semantics and platform, nor progressive enhancements.

An element that issues à GET and replaces the current page is obviously primarily an anchor. Yet HTMX wouldn’t work with a bare anchor. OTOH, Turbo would. I think Turbo is better designed.